docs/feature_split_keyboard.md

```diff
@@ -133,6 +133,12 @@ However, you'll have to flash the EEPROM files for the correct hand to each cont
 * `:dfu-util-split-left`
 * `:dfu-util-split-right`
 
+Example:
+
+```c
```

Comment:
‘'c' is not required

Reply:
Thanks @mtei. I'm aware but I left it there for _consistency_ with the rest of the document. Let me know if I should still remove it for this one.

Reply:
This is not C code, so it should be removed.

Reply:
Edited as requested.